joomla/joomla-cms
# [3.9] convert `JToolbarHelper` to a namespaced class name

### Summary of Changes

`JToolbarHelper` is now `Joomla\CMS\Toolbar\ToolbarHelper`. This change is necessary to further the adoption of namespaced naming schemes as opposed to the alternative old style.

### Testing Instructions

1. Check for any change in behavior of code that is presently using the `JSubMenuHelper` class name.
2. Check for any change in behavior of code that is presently using the `JToolbarHelper` class name.
3. Change an instance of `JToolbarHelper` to `Joomla\CMS\Toolbar\ToolbarHelper` in some arbitrary code and check for any change in behavior.

### Expected result

No change in behavior should occur as a result of the outlined testing methodology. The use of `JToolbarHelper` should continue to work like normal since it has been aliased to the namespaced edition of the class. The functionality of the namespaced edition of the class should be identical to that of the old version.

Since `JToolbarHelper` registered `JSubMenuHelper` with `JLoader`, `administrator/index.php` now requires `subtoolbar.php` so that the class can be made available. This was the preferred method of action as opposed to converting it to a namespaced variant since the class is deprecated.

### Documentation Changes Required

Yes. Any recommendation of `JToolbarHelper` should be changed in favor of using `Joomla\CMS\Toolbar\ToolbarHelper`.
